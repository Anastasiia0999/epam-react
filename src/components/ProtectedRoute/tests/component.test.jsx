import React from 'react';
import { mount, shallow } from 'enzyme';

import {ProtectedRoute} from "../index";


describe('<ProtectedRoute />', () =>{
    let wrapper;
    let instance;

    const props = {
        logStatus: false
    };

    beforeEach(() => {
        wrapper = shallow(<ProtectedRoute {...props} />);
        instance = wrapper.instance();
    });

    it('should render correctly', () =>{
        expect(wrapper.html()).toMatchSnapshot();
    });


});


