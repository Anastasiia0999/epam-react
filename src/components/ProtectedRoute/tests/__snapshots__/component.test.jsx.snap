// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProtectedRoute /> should render correctly 1`] = `"<div></div>"`;
